.github/workflows/linux-cuda-gnu.yml

```diff
@@ -76,13 +76,14 @@ jobs:
       # evision related env vars
       MIX_ENV: test
       OPENCV_VER: "4.9.0"
-      OTP_VERSION: "25.2"
-      ELIXIR_VERSION: "1.14.3"
+      OTP_VERSION: "25.3"
```

Comment:
The https://repo.uwucocoa.moe/pool/main/ has no `erlang_25.3.*` package. 😅

Reply:
Hi @yuriploc, thanks for the comment! I've changed it to use precompiled OTP from cocoa-xu/otp-build. :)